apps/meteor/app/slashcommands-open/client/client.ts

```diff
@@ -1,46 +1,47 @@
 import { Meteor } from 'meteor/meteor';
 import { FlowRouter } from 'meteor/kadira:flow-router';
-import type { IMessage } from '@rocket.chat/core-typings';
 
 import { roomCoordinator } from '../../../client/lib/rooms/roomCoordinator';
 import { slashCommands } from '../../utils/lib/slashCommand';
 import { Subscriptions, ChatSubscription } from '../../models/client';
 
-function Open(_command: 'open', params: string, _item: IMessage): void {
-	const dict: Record<string, string[]> = {
-		'#': ['c', 'p'],
-		'@': ['d'],
-	};
+slashCommands.add(
+	'open',
+	function Open(_command, params): void {
```

Comment:
Nesting this function declaration as a parameter expression damages readability, as evidenced by the indentation increase.